components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/util/TemplateManagerConstants.java

```diff
@@ -18,19 +18,23 @@
 
 package org.wso2.carbon.business.rules.core.util;
 
+import org.wso2.carbon.utils.Utils;
+
 /**
  * Consists of constants related to root.Template Manager
  */
 public class TemplateManagerConstants {
     // Directory locations
-    public static final String TEMPLATES_DIRECTORY = "/home/senthuran/Desktop/rough-templates/"; // todo: not finalized
+//    public static final String CARBON_HOME = Utils.getCarbonHome().toString();
+    //    public static final String TEMPLATES_DIRECTORY = CARBON_HOME+"/resources/businessRules/templates/"; // todo: not
```

Comment:
Is this necessary?

Reply:
This is the required one. I just commented it to test the method locally without packing with SP

Reply:
@AnuGayan, shall we remove this?

Reply:
This was fixed by 47e9832228beaa5747e074303ac9d729cc69abdf and  966ec96dbd7f65065ffae47d756cafd9ef841718